### PR TITLE
Reword explanation of ClusterIP for Services

### PR DESCRIPTION
**Pull request by guettli**
It took me some time to understand that.

Feel free to change my wording

### CODE REVIEW

**Comment by sftim** [comment]
/retitle Reword explanation of ClusterIP for Services

File: content/en/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -598,7 +598,8 @@ The default is `ClusterIP`.
 `Type` values and their behaviors are:
 
 * `ClusterIP`: Exposes the Service on a cluster-internal IP. Choosing this value
-  makes the Service only reachable from within the cluster. This is the
+  makes the Service only reachable from within the cluster. But you can expose
+  the service to the public with an Ingress. This is the
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/service.md:602]: Maybe we could also mention [Gateway](gateway-api.sigs.k8s.io/) here?

**Comment by guettli** [comment]
@sftim I added "Gateway API". Is it ok like this?

**Review by Arhell — COMMENTED** [review]
Please sign CLA

**Comment by sftim** [comment]
@guettli please check the commit email address that you're using, it might not match the one you signed the CLA with.

**Comment by guettli** [comment]
I signed it

**Comment by guettli** [comment]
I am happy, the EasyCLA now passed. There was some confusion with an outdated email address.

**Comment by guettli** [comment]
@Arhell @sftim could you please review this tiny PR?

File: content/en/docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -653,6 +653,8 @@ Kubernetes `ServiceTypes` allow you to specify what kind of Service you want.
 * `ClusterIP`: Exposes the Service on a cluster-internal IP. Choosing this value
   makes the Service only reachable from within the cluster. This is the
   default that is used if you don't explicitly specify a `type` for a Service.
+  You can expose the service to the public with an Ingress or the
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/services-networking/service.md:656]:
```suggestion
  You can expose the service to the public with an [Ingress](docs/reference/kubernetes-api/service-resources/ingress-v1/) or the
```

File: content/en/docs/concepts/services-networking/service.md [hunk 3]
```diff
@@ -653,6 +653,8 @@ Kubernetes `ServiceTypes` allow you to specify what kind of Service you want.
...
+  Gateway API.
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/services-networking/service.md:657]:
```suggestion
  [Gateway API](https://gateway-api.sigs.k8s.io/).
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Hyperlinked the two terms for the sake of clarity. Rest LGTM from the docs perspective!

**Review by sftim — COMMENTED** [review]
/lgtm
from https://github.com/kubernetes/website/pull/37330#pullrequestreview-1251348126

/approve

File: content/en/docs/concepts/services-networking/service.md [hunk 4]
```diff
@@ -653,6 +653,8 @@ Kubernetes `ServiceTypes` allow you to specify what kind of Service you want.
 * `ClusterIP`: Exposes the Service on a cluster-internal IP. Choosing this value
   makes the Service only reachable from within the cluster. This is the
   default that is used if you don't explicitly specify a `type` for a Service.
+  You can expose the service to the public with an [Ingress](docs/reference/kubernetes-api/service-resources/ingress-v1/) or the
+  [Gateway API](https://gateway-api.sigs.k8s.io/).
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/service.md:657]: IMO this is still not quite right, but it's still a helpful change to make.

**Comment by tengqm** [comment]
The link target misses a leading `/`, this needs a fix.

**Comment by sftim** [comment]
Oops. https://github.com/kubernetes/website/pull/39185 should fix that.

**Pull request merged**